scss/window/_theme.scss

```diff
@@ -18,14 +18,17 @@
     // Titlebar
     .k-window-titlebar {
         @include appearance( header );
+        color: $window-titlebar-text;
         border-color: inherit;
+        background: $window-titlebar-bg;
```

Comment:
Since we are overriding the appearance, we don't really need `appearance(header)`. In terms, this could become:

```scss
.k-window-titlebar {
    @include fill( $window-titlebar-text, $window-titlebar-bg, $window-titlebar-border);
}
// or, the longer form
.k-window-titlebar {
    border-color: $window-titlebar-border;
    color: $window-titlebar-text;
    background-color: $window-titlebar-bg;
}